eva-accession-release-automation/gather_clustering_counts/gather_per_species_clustering_counts.py

```diff
@@ -92,6 +92,9 @@ def gather_counts(private_config_xml_file, release_version, release_dir):
         taxid, sci_name = get_taxonomy_and_scientific_name(private_config_xml_file, release_version, species_dir)
         if not taxid or not sci_name:
             continue
+        if "'" in sci_name:
+            # some scientific names in db has single quote (') in name causing issues while inserting
+            sci_name = sci_name.replace("'", "\''")
```

Comment:
We should use the same [normalisation process](https://github.com/EBIvariation/eva-common-pyutils/blob/927f6116319a6ad29ba1a8bbd00224eac19b1f11/ebi_eva_common_pyutils/taxonomy/taxonomy.py#L30) as elsewhere.

Reply:
I misunderstood the purpose of this change. appologies